mobile_app_connector/models/__init__.py

```diff
@@ -10,4 +10,5 @@
 ##############################################################################
 
 from . import compassion_child
+from . import compassion_donation
```

Comment:
Rename the file `product_template`, it's better to name the file following odoo model to easily know what it's about